nucliadb_protos/python/nucliadb_protos/nodereader_pb2_grpc.pyi

```diff
@@ -3,10 +3,10 @@
 isort:skip_file
```

Comment:
Is it normal to commit those `*.pyi` files ?

Reply:
Yes, are part of the type checking for Python. I think we commit them for people using nucliadb_protos package. @bloodbare correct me if I'm wrong